docs/README.md

```diff
@@ -45,6 +45,8 @@ External Links:
     https://en.wikipedia.org/wiki/Percent-encoding  
     https://tools.ietf.org/html/rfc3986#page-11
 
+For multi-level tlds such as `example.co.uk`, "co" is treated as a subzone from a DNS perspective and would be used as {label1}
```

Comment:
{label1} would be `example.com` in this case or?
{label2} would be `co` and {label3} would be `uk`

Reply:
Right, I changed the wording to accurately reflect how this example would be handled. I also changed the inline comment.